vuln/npm/366.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": 366,
+  "created_at": "2018-01-23",
+  "updated_at": "2018-01-23",
+  "title": "html-janitor passing user-controlled data to clean() leads to XSS",
+  "author": "_bayotop (https://twitter.com/_bayotop)",
+  "module_name": "html-janitor",
+  "publish_date": "2018-01-23",
+  "cves": [
+    "CVE-2017-0931"
+  ],
+  "vulnerable_versions": "<=2.0.2",
+  "patched_versions": "<0.0.0",
```

Comment:
Can you place an empty string here? (as of https://github.com/nodejs/security-wg/blob/master/vuln/npm/361.json#L11)